index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href=".css">
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
```

Comment:
Great job @Ibtisam34 🤝

- [ ] [Optional] As suggested on project requirements, kindly add a class to your element and then use this class in your stylesheet to apply the styles on this element.